### PR TITLE
Drop old things

### PR DESCRIPTION
**Pull request by necessarily-equal**
Comes along https://github.com/DaemonEngine/Daemon/pull/518 but doesn't depend on it.

### CODE REVIEW

**Comment by slipher** [comment]
Inline linkage should never be used in a .cpp. So all of these `INLINE`s should be deleted rather than replaced with `inline`.

**Comment by necessarily-equal** [comment]
I've dropped the `inline` in cpp files. I don't think they are very useful, even if I think they didn't hurt either.

File: src/sgame/sg_bot.h [hunk 1]
```diff
@@ -48,12 +48,7 @@ struct enemyQueueElement_t
 	int        timeFound;
 };
 
-struct enemyQueue_t
-{
-	enemyQueueElement_t enemys[ MAX_ENEMY_QUEUE ];
-	int front;
-	int back;
-};
+using enemyQueue_t = Queue<enemyQueueElement_t, MAX_CLIENTS+1>;
```
Comment (slipher) [inline src/sgame/sg_bot.h:51]: compile error here

**Comment by necessarily-equal** [comment]
whoops, I didn't mean to commit this.

**Pull request merged**